stepLogger_cucumber_plugin/src/main/java/com/github/bogdan5555556/CucumberStepsConsoleLogger/CucumberStepsConsoleLogger.java

```diff
@@ -128,7 +138,17 @@ public void match(Match match) {
             stepLogName = stepLogName + "\n";
             stepLogName = stepLogName + "\'\'\'" + "\n" + step.getDocString().getValue() + "\n" + "\'\'\'";
         }
-        LOGGER.info("\n\n---------- Test Step Execution ----------\n" + ANSI_MAGENTA + stepLogName + ANSI_RESET + "\n---------- Test Step Execution ----------\n");
+        LOGGER.info(buildStepExecutionMessageBuild(stepLogName));
+    }
+
+    private String buildStepExecutionMessageBuild(String stepLogName){
```

Comment:
Looks like there is a duplicate in method's name: `buildStepExecutionMessageBuild`. Use just `buildStepExecutionMessage`.